docs/hyn/5.4/spatie-laravel-permission.md

```diff
@@ -67,20 +68,20 @@ class Role extends SpatieRole
 }
 ```
 
-`App\User.php` - notice the `HasRoles`, `UsesTenantConnection` traits we added.
+`App\User.php` - notice the `HasRoles`, `UsesSystemConnection` traits we added.
 ```php
 <?php
 
 namespace App;
 
 use Spatie\Permission\Traits\HasRoles;
 use Illuminate\Notifications\Notifiable;
-use Hyn\Tenancy\Traits\UsesTenantConnection;
+use Hyn\Tenancy\Traits\UsesSystemConnection;
```

Comment:
Not everyone has their users on the system database.  I think it's fine to leave this as-is, and users can change it to meet their needs